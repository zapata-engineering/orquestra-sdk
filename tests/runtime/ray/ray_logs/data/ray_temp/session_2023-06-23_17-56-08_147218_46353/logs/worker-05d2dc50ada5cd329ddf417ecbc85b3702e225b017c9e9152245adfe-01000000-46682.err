:job_id:01000000
:task_name:_workflow_task_executor_remote
2023-06-23 17:56:50,870	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.1cfe045@invocation-1-task-add-with-log]
ORQ-MARKER:{"event": "task_start", "timestamp": "2023-06-23T17:56:50.875741+02:00", "wf_run_id": "wf.wf_using_python_imports.1cfe045", "task_inv_id": "invocation-1-task-add-with-log"}
ORQ-MARKER:{"event": "task_end", "timestamp": "2023-06-23T17:56:50.876941+02:00", "wf_run_id": "wf.wf_using_python_imports.1cfe045", "task_inv_id": "invocation-1-task-add-with-log"}
:task_name:_workflow_task_executor_remote
2023-06-23 17:57:41,140	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.1cfe045@orquestra.sdk._ray._build_workflow._make_ray_dag_node.locals._ray_remote]
:task_name:_workflow_task_executor_remote
2023-06-23 17:57:41,151	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.1cfe045@orquestra.sdk._ray._build_workflow.make_ray_dag.locals.handle_data_aggregation_error]
