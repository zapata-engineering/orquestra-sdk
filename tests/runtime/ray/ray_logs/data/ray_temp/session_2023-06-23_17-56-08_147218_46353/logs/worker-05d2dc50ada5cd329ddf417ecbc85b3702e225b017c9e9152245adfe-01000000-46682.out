:job_id:01000000
:task_name:_workflow_task_executor_remote
ORQ-MARKER:{"event": "task_start", "timestamp": "2023-06-23T17:56:50.875741+02:00", "wf_run_id": "wf.wf_using_python_imports.1cfe045", "task_inv_id": "invocation-1-task-add-with-log"}
hello, there!
ORQ-MARKER:{"event": "task_end", "timestamp": "2023-06-23T17:56:50.876941+02:00", "wf_run_id": "wf.wf_using_python_imports.1cfe045", "task_inv_id": "invocation-1-task-add-with-log"}
:task_name:_workflow_task_executor_remote
:task_name:_workflow_task_executor_remote
